### ISSUE TITLE
Possessed Hag nerf

### ISSUE THREAD

**Issue by WightofShadows** [TheCyberBrick/The-Betweenlands#642]
Putting this here mainly because most other people seem to agree that they’re too powerful atm. Haven’t had the chance to look at them myself though, so this will need feedback from people who can propose possible solutions, if they even need one.

EDIT: And please, do propose solutions, _**don't just complain**_.

**Comment by PetasiGit** [comment]
The main issue is the spirit things. They can target you from so far away that sometimes it's even difficult to take out a turret with a bow. There's also a lot of them, so maybe reduce the number and targeting distance by a third or so?

**Comment by TripleHeadedSheep** [comment]
Really unsure about this so far. I had a few test runs with them and I do agree they offer a challenge. But when using mid- to late-game equipment, they are pretty do-able. In my opinion they offer a good challenge, but you should avoid them when unprepared. It adds to the difficulty of the Betweenlands, and reminds the player that they cannot simply dominate this world :P

**Comment by alxcool** [comment]
When I am in a cavern... Maybe suggest the player to bring some crafted blocks? Betweenstone and pitstone, the IA can see through them like if its was air. But, Dirt, Planks, and probably many more like Cragrock, Mobs doesn't seems to be able to follow the player's movements so easily.

As suggestion, A new Structure like a  Betweenlands equivalent of vanilla Mineshaft, could help players to hide for a bit.

When on surface, well run far away, heal then plan your next move! 

Missile attack... Give a %chance to deflect them using a shield... They are not that deathly but are quite frustrating. When I run away, they keep hovering around me, and I never know when they would hit. So far I'm not sure if this work, but using a shield trying to block them, seems to delay them. They still hit me 100% for 3-4 damage, but take twice more time.

% Possession failure "That's not Wight!" ... For the Lore let say, the Hag get annoyed by his Wight. Give a % to separate both entity after 1min... every 2 sec maybe...
...
Splash, then Knockback... Something annoying I've encounter is having more than 7 hag + 1 Wight running after me in a well lit cavern. So When a wight try to possess a Hag, that could help the player if it create a Splash potion effect similar to knockback, around the targeted Hag. That way, the others would be confuse for 1 sec!

Careful when playing with IA! In some others mods, I've encountered some issue. As example when the Mob enter a state who was not planned... Like as example When a skeleton hit an another skeleton, both of them will start fighting each one another. But when it come to modded mobs, sometime such events are not well programmed or planned, then once the hostile mob kill the ranged one, he become idle.

**Comment by MrRedstoneA1** [comment]
They definitely need to be nerfed. Remember, the spirit projectiles bypass any armor. They can literally chase you forever if you don't outrun them. I tested it, they chase you forever until you either die or outrun them. You only have about 5-10 seconds to either kill the wight or outrun the projectiles before death.

**Comment by TripleHeadedSheep** [comment]
> They definitely need to be nerfed. Remember, the spirit projectiles bypass any armor. They can literally chase you forever if you don't outrun them. I tested it, they chase you forever until you either die or outrun them. You only have about 5-10 seconds to either kill the wight or outrun the projectiles before death.

I disagree. We've had them as they are for a while now and they seem to be perfectly fine for the game as of now. They are not a very common occurence and if they do happen a player still has a brief moment to see it coming. If you're not ready to take it on, running is always an option.
Hag turrets will remain a challenge, even in late game, but there is no point to making everything save for a late game run. They are the required 'oh shit' moment to keep encounters interesting.
Besides, if you get the ancient armour you can block 50% of a Wight's possession damage anyways ;)
